daprdocs/content/en/developing-ai/dapr-agents/dapr-agents-getting-started.md

```diff
@@ -51,162 +51,326 @@ docker ps
 Make sure you have Python already installed. `Python >=3.10`. For installation instructions, visit the official [Python installation guide](https://www.python.org/downloads/).
 {{% /alert %}}
 
-## Create Your First Dapr Agent
+## Prepare your environment
 
-Let's create a weather assistant agent that demonstrates tool calling with Dapr state management used for conversation memory.
+In this getting started guide, you’ll work directly from the [Dapr Agents' quickstarts](https://github.com/dapr/dapr-agents/tree/main/quickstarts). We’ll focus on the **`06_durable_agent_http.py`** example, which is a reliable durable agent implemented with Dapr’s workflow engine and exposed over HTTP.
 
-### 1. Create the Dapr components
+### 1. Clone the repository and examine its content
 
-Create a `components` directory and add two files:
+```bash
+git clone https://github.com/dapr/dapr-agents.git
+cd dapr-agents/quickstarts/01-dapr-agents-fundamentals
+```
+
+### 2. Create a virtual environment and install dependencies
+
+From the `01-dapr-agents-fundamentals` folder, do:
+
+```bash
+python3.10 -m venv .venv
+
+# Activate the virtual environment 
+# On Windows:
+.venv\Scripts\activate
+# On macOS/Linux:
+source .venv/bin/activate
+
+# Install dependencies from the quickstart
+pip install -r requirements.txt
+```
 
-`historystore.yaml`:
+This installs `dapr-agents` and any additional libraries needed by the examples.
+
+## Understand the application
+
+This example creates an agent that assists with weather information and uses Dapr to handle LLM interactions, persist conversation history, and provide reliable, durable execution of the agent’s steps.
+
+For this quickstart you’ll primarily work with:
+
+* `06_durable_agent_http.py` – the main durable weather agent application exposed over HTTP
+* `function_tools.py` – contains `slow_weather_func`, the tool used by the agent
+* `resources/llm-provider.yaml` – Conversation API and LLM configuration
+* `resources/conversation-statestore.yaml` – conversation memory state store
```

Comment:
Should this be called conversation or just memory store? And where does the term session come into this?